src/labels_by_user_input.py

```diff
@@ -26,23 +29,19 @@ def labels_by_user_input(data, pull):
     body = data["comment"]["body"]
     issue_labels = get_labels(pull=pull)
     comment_labels = re.findall("/\\S*", body)
-    print(body)
-    print(comment_labels)
 
     for label in comment_labels:
         label = label.lstrip("/").lower()
```

Comment:
Please check here if label is not owned by us

```
label = label.lstrip("/").lower()
unlabel = label.startswith("un")
unlabel_name = label.lstrip("un") if unlabel else None

if label not in LABELS_SET:
    print(UNSUPPORTED_LABELS)
    return pull.create_comment(body=UNSUPPORTED_LABELS)
```

Reply:
can we make a nice comment (title and so on)?

Reply:
@myakove There's already a check to see whether a label is owned by us - 
`if stripped_label in LABELS_SET:`
I didn't add the 'unlabel' bool as it just adds a check in the situation there's a need to unlabel,
Even though this can be checked once for both situations